.yarnrc

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+email a@snd.rs
+lastUpdateCheck 1633433443971
+username sndrs
```

Comment:
Heh. Does this file need your email and GitHub username?

Reply:
oh wow how wierd! i just ran `yarn policies set-version` and committed. i don't think that's important info no :)

Reply:
but then... `# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.`

will it just keep appearing with the last editor's info?

Reply:
that email is wrong too :(

Reply:
i had some yarn config setup for some reason, removed and reran